### PR TITLE
[netem] minimalist support for netem

### PR DESCRIPTION
**Pull request by chantra**
Support for tc_netem_qopt options, e.g: latency, limit, loss, gap, duplicate and jitter

### CODE REVIEW

File: qdisc.go [hunk 1]
```diff
@@ -120,6 +125,73 @@ func (qdisc *Htb) Type() string {
 	return "htb"
 }
 
+// Netem is a classless qdisc that rate limits based on tokens
+
+type NetemQdiscAttrs struct {
+	Latency   uint32 // in us
```
Comment (zfjagann) [inline qdisc.go]: Maybe consider using [time.Duration](https://golang.org/pkg/time/#Duration)?

Comment (chantra) [inline qdisc.go]: I am not sure it is worth it given that it will need to be casted back to a uint32 in netlink.

Comment (zfjagann) [inline qdisc.go]: The reason being time.Duration is unit agnostic and more go idiomatic. See #56

File: qdisc_test.go [hunk 2]
```diff
@@ -88,13 +88,6 @@ func TestHtbAddDel(t *testing.T) {
 		t.Fatal(err)
 	}
 
-	/*
-		cmd := exec.Command("tc", "qdisc")
-		out, err := cmd.CombinedOutput()
-		if err == nil {
-			fmt.Printf("%s\n", out)
-		}
-	*/
```
Comment (vishvananda) [inline qdisc_test.go]: thanks for removing this debug code

File: qdisc_linux.go [hunk 3]
```diff
@@ -166,6 +201,16 @@ func QdiscList(link Link) ([]Qdisc, error) {
 					if err := parseHtbData(qdisc, data); err != nil {
 						return nil, err
 					}
+				case "netem":
+					/*
+						data, err := nl.ParseRouteAttr(attr.Value)
+						if err != nil {
+							return nil, err
+						}
+					*/
```
Comment (vishvananda) [inline qdisc_linux.go]: this looks like debug code. Do you mind removing it?

**Comment by vishvananda** [comment]
looks good. Small suggestion inline

**Comment by chantra** [comment]
I `git push --force` with the comment removed.

**Pull request merged**